### PR TITLE
[indexer-alt] Add processor lag metrics

### PR DESCRIPTION
**Pull request by lxfind**
## Description 

Describe the changes or additions included in this PR.

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

**Review by amnn — APPROVED** [review]
LGTM, thanks @lxfind, just a nit about import order.

Do you imagine you'll need to add this kind of metric in more places? If so, it's verging on it being useful to create a special kind of `IntGauge` to expose `max` API (it seems a shame to have to maintain a separate `AtomicU64` when each metric is its own atomic counter).

**Comment by lxfind** [comment]
> If so, it's verging on it being useful to create a special kind of IntGauge to expose max API

It is indeed attempting, but the ingestion metrics are non-vector metrics, while the processor/committer metrics are vector metrics. So it was difficult to share between them.

File: crates/sui-indexer-alt/src/metrics.rs [hunk 1]
```diff
@@ -1,8 +1,6 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-use std::{net::SocketAddr, sync::Arc};
```
[the remark this answers is not in the document]
Comment (lxfind) [reply crates/sui-indexer-alt/src/metrics.rs:4]: hmm are you using vs code? RustRover would put them all together and sort by name.

[the remark this answers is not in the document]
Comment (amnn) [reply crates/sui-indexer-alt/src/metrics.rs:4]: I use vim, but the imports are handled the same as VSCode, (by the LSP). I wouldn't be surprised if RustRover implements this differently though (based on the in-editor intelligence).

**Pull request merged**